cli/internal/run/dry_run.go

```diff
@@ -215,6 +240,23 @@ func displayDryTextRun(ui cli.Ui, summary *dryRunSummary, workspaceInfos graph.W
 		}
 	}
 
+	fileCount := 0
+	for range summary.GlobalHashSummary.GlobalFileHashMap {
+		fileCount = fileCount + 1
+	}
+	w1 := tabwriter.NewWriter(os.Stdout, 0, 0, 1, ' ', 0)
+	ui.Output("")
+	ui.Info(util.Sprintf("${CYAN}${BOLD}Global Hash Inputs${RESET}"))
+	fmt.Fprintln(w1, util.Sprintf("  ${GREY}Global Files\t=\t%d${RESET}", fileCount))
```

Comment:
added a length here instead of the full list (json has the full list), since it could be really large. i'm not sure anyone will care about the full expanded list except to try to parse/operate on it

Reply:
Always something we could revisit later if needed.